dataverse-webapp/src/test/java/edu/harvard/iq/dataverse/util/ShapefileHandlerTest.java

```diff
@@ -167,6 +210,9 @@ private void addToZipFile(String fileName, byte[] fileAsBytesToZip, ZipOutputStr
         zipOutputStream.closeEntry();
     }
 
+    private @NotNull ShapefileHandler newShapeFileHandler(File zipfile_obj) {
```

Comment:
Most likely in some environments `@NotNull` from `org.jetbrains.annotations` will not be available.

Reply:
The dependency `org.jetbrains:annotations` is brought into `test` scope by `org.testcontainers:testcontainers` and intellij makes use of it during code generation if present in classpath. But doesn't provide much in this case, so I removed it.